packages/node/src/integrations/localvariables/inspector.d.ts

```diff
@@ -3357,3 +3357,7 @@ declare module 'node:inspector' {
   import inspector = require('inspector');
   export = inspector;
 }
+
+declare module 'node:inspector/promises' {
+  //
```

Comment:
Can you add a comment to here? What's the purpose of these lines? When should we remove it etc?